ci-operator/config/stackrox/collector/stackrox-collector-mauro-refactor-stackrox-collector-config.yaml

```diff
@@ -266,32 +270,11 @@ resources:
       memory: 2Gi
 tests:
 - as: lint
-  steps:
-    test:
-    - as: clang-format
-      commands: make -C collector check
-      from: src
-      resources:
-        requests:
-          cpu: 2000m
-          memory: 4000Mi
-    - as: shfmt-check
-      commands: make shfmt-check
-      from: src
-      resources:
-        requests:
-          cpu: 2000m
-          memory: 4000Mi
-- as: test-scripts
-  steps:
-    test:
-    - as: shellcheck-all
-      commands: make shellcheck-all
-      from: src
-      resources:
-        requests:
-          cpu: 2000m
-          memory: 4000Mi
+  commands: |
```

Comment:
[nit] somewhat outside the scope of this PR but it might be worth making this a required step

Reply:
I agree, but the original lints were not defined as required, so we might want to do it in a follow up